pkg/test/config/toolchainconfig.go

```diff
@@ -49,6 +49,22 @@ func PerMemberCluster(name string, value int) PerMemberClusterOptionInt {
 	}
 }
 
+//---PublicViewer Configuration---//
```

Comment:
very minor thing - now it's part of the Host configuration right ?

Reply:
great catch! let me fix this